examples/tutorials/analysis-3d/non-detection.py

```diff
@@ -0,0 +1,346 @@
+# -----------
+#
+
+from gammapy.utils.check import check_tutorials_setup
+
+check_tutorials_setup()
+
+
+######################################################################
+# When we don’t detect a source
+# -----------------------------
+#
+# As usual, let’s start with some general imports…
+#
+
+
+######################################################################
+# Now, load and inspect the dataset. For computational purposes, we have
+# already created a `MapDataset` from observation id 20275 from the
+# public H.E.S.S. data release and stored it in `GAMMAPY_DATA`
+#
+
+dataset = MapDataset.read("$GAMMAPY_DATA/empty_dl4/empty-dl4.fits.gz")
+dataset.peek()
+plt.show()
+
+
+######################################################################
+# We will first use the `ExcessMapEstimator` for a quick check to see if
+# there are any hotspots in the field. You may also use the
+# `TSMapEstimator`.
+#
+
+estimator = ExcessMapEstimator(sum_over_energy_groups=True, selection_optional="all")
+res1 = estimator.run(dataset)
+res1.sqrt_ts.plot(add_cbar=True)
+plt.show()
+
+
+######################################################################
+# The significance map looks rather flat! Suppose we were expecting a
+# source at the centre of our map. Lets try see if we can fit a point
+# source there.
+#
+
+spectral = PowerLawSpectralModel()
+spatial = PointSpatialModel(frame="icrs")
+spatial.lon_0.value = 187.0
+spatial.lat_0.value = 2.6
+spatial.lat_0.min = 1.0
+spatial.lat_0.max = 4.0
+spatial.lon_0.min = 185
+spatial.lon_0.max = 187
+
+skymodel = SkyModel(spatial_model=spatial, spectral_model=spectral, name="test")
+
+dataset.models = skymodel
+fit = Fit()
+res = fit.run(dataset)
+print(res.models)
+
+
+######################################################################
+# We can see that there is a slight negative excess in the centre, and
+# thus, the fitted model has a negative amplitude. We can use the
+# `select_nested_models` function to perform a likelihood ratio test to
+# see if this is significant.
+#
+
+LLR = select_nested_models(
+    Datasets(dataset), parameters=[skymodel.parameters["amplitude"]], null_values=[0]
+)
+print(LLR)
+
+
+######################################################################
+# You can see that the `ts ~ 4.7`, thus suggesting that the observed
+# fluctuations are not significant over the background. Now, we will
+# estimate the differential upper limits of the source.
+#
+# Here, it is important to **set a reasonable model** on the dataset
+# before proceeding with the estimator. This model can come from
+# measurements from other instruments, be an extrapolation of the flux
+# observed at other wavelengths, come from theoretical estimations, etc. A
+# model with a negative amplitude as obtained above should should not be
+# used.
+#
+# Note that the computed upper limits can depend on the assumed model. The
+# values of the amplitude should not matter as long as the values are not
```

Comment:
I do not understand this sentance

Reply:
What I meant is that the computed UL can differ depending on the source model, say a power law vs an exponential cutoff. Since the value of the normalisation is adjusted in each bin, the ULs generally dont change with the amplitude value, unless of course one puts a value very far away from reality. I will try to rephrase

Reply:
Sorry, my question was not clear... I was speaking about:
`The values of the amplitude should not matter as long as the values are not too absurd.`
I think that the sentence is correct, but not well placed... Maybe
1. should your spectral shape with an index
2. then you can place your sentence about the normalisation